cmd/kubeadm/app/apis/kubeadm/constants/BUILD

```diff
@@ -1,15 +1,10 @@
-package(default_visibility = ["//visibility:public"])
-
-load(
-    "@io_bazel_rules_go//go:def.bzl",
-    "go_library",
-    "go_test",
-)
+load("@io_bazel_rules_go//go:def.bzl", "go_library", "go_test")
 
 go_library(
     name = "go_default_library",
     srcs = ["constants.go"],
-    importpath = "k8s.io/kubernetes/cmd/kubeadm/app/constants",
+    importpath = "k8s.io/kubernetes/cmd/kubeadm/app/apis/kubeadm/constants",
+    visibility = ["//visibility:public"],
```

Comment:
is this an automated change?

Reply:
We didn't knowingly add that manually ... let me check ...

Reply:
ah ok, no worries, was gong to suggest moving this to default visibility, but i prefer whatever the automated output is

Reply:
Deleting `cmd/kubeadm/app/apis/kubeadm/constants/BUILD` and running `./hack/update-bazel.sh` generates the same file (some reordering, but no change of the `visibility` setting for `go_default_library`).
So, yes, automated change.